WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -93,23 +94,26 @@ public void OnWalletOpened()
 		{
 			Disposables = Disposables is null ? new CompositeDisposable() : throw new NotSupportedException($"Cannot open {GetType().Name} before closing it.");
 
-			var observed = Observable.Merge(
-				Global.UiConfig.WhenAnyValue(x => x.LurkingWifeMode).Select(_ => Unit.Default),
+			Observable.Merge(
 				Observable.FromEventPattern(Global.WalletService.TransactionProcessor, nameof(Global.WalletService.TransactionProcessor.CoinReceived)).Select(_ => Unit.Default),
 				Observable.FromEventPattern(Global.WalletService.TransactionProcessor, nameof(Global.WalletService.TransactionProcessor.CoinSpent)).Select(_ => Unit.Default))
-				.Throttle(TimeSpan.FromSeconds(1))
-				.ObserveOn(RxApp.MainThreadScheduler);
-
-			observed.Subscribe(_ =>
+				.Throttle(TimeSpan.FromSeconds(0.5))
```

Comment:
Why throttle here? Why not before `ObserveOn`?

Reply:
Let me chime in because we did this together. It is before `ObserveOn`. I was bothered with the slow reaction of the amount change when clicking on it. So we moved the LWM mode trigger after the Throttle. With this the reaction time of LWM is fast and CoinChanges still Throttled with half seconds.

Reply:
Because when you click on LWM we want to see the visual feedback immediately and not after a second (or half a second as this PR).